airbyte-cdk/python/unit_tests/sources/file_based/scenarios/unstructured_scenarios.py

```diff
@@ -74,6 +74,7 @@
                     },
                     "name": "stream1",
                     "source_defined_cursor": True,
+                    'source_defined_primary_key': [['document_key']],
```

Comment:
Just FYI, some linters will require double-quotes here for consistency.

https://docs.astral.sh/ruff/rules/bad-quotes-inline-string/

That said, this is an easy auto-fix when we get Ruff setup in the repo, so I'm not worried about it per se.